common-pg/src/main/scala/com/socrata/pg/store/SecondarySchemaLoader.scala

```diff
@@ -140,18 +140,22 @@ class SecondarySchemaLoader[CT, CV](conn: Connection, dsLogger: Logger[CT, CV],
   protected def dropFullTextSearchIndex(columnInfos: Iterable[ColumnInfo[CT]]): Unit =
     if (columnInfos.nonEmpty) {
       val table = tableName(columnInfos)
-      using(conn.createStatement()) { (stmt: Statement) =>
-        stmt.execute(s"DROP INDEX IF EXISTS idx_search_${table}")
+      using(conn.prepareStatement(pendingIndexDropSql)) { stmt =>
+        stmt.setString(1, s"idx_search_${table}")
+        stmt.executeUpdate()
       }
     }
 
   protected def dropIndexes(columnInfos: Iterable[ColumnInfo[CT]]): Unit = {
     val table = tableName(columnInfos)
-    using(conn.createStatement()) { stmt =>
+    using(conn.prepareStatement(pendingIndexDropSql)) { stmt =>
       for {
         ci <- columnInfos
-        indexSql <- repFor(ci).dropIndex(table)
-      } stmt.execute(indexSql)
+        dbIndexName <- repFor(ci).dbIndexNames(table)
```

Comment:
dbIndexNames is plural, might it return more than one val?

Reply:
Yes, a column has can have more than one standard indexes.